front_backpackers/src/App.js

```diff
@@ -4,116 +4,53 @@ import Login from "./Login";
-        },
-        {
-          idPlace: 3,
-          name: "Meltdown Reims",
-          city: "Reims",
-          adress: "188 Rue de Vesle",
-          price: 5,
-          type: "bar e-sport",
-          capacity: 30,
-          description:
-            "Bar e-sport ou vous pouvez suivre les compétitions e-sport tout en buvant une bonne bière !",
-          contact: "MeltdownReims@gmail.com, 07.78.32.45.88",
-          rating: null,
-          picture: "https://zupimages.net/up/18/48/y2ww.png"
-        }
-      ],
-      activityList: [
-        {
-          idActivity: 1,
-          idPlace: 1,
-          city: "Reims",
-          name: "Soirée au Dropkick",
-          price: 15,
-          type: "rencontre",
-          capacity: 10,
-          picture: "https://zupimages.net/up/18/48/7hk0.jpg",
-          description:
-            "J'organise une soirée pour faire de nouvelle rencontre car je suis nouveau dans cette ville",
-          contact: "06.43.56.32.78",
-          rating: null,
-          creator: "Michael",
-          participants: null,
-          date: "6 décembre 2018"
-        },
-        {
-          idActivity: 2,
-          idPlace: 2,
-          city: "Reims",
-          name: "Aquaman (imax)",
-          price: 16.3,
-          type: "cinéma",
-          capacity: 6,
-          picture: "https://zupimages.net/up/18/48/4yyh.png",
-          description:
-            "J'organise une sortie au cinéma pour aller voir Aquaman en Imax au Gaumont Parc Millésime, on verras sur place si on se fait une bouffe après !",
-          contact: "06.67.98.56.21",
-          rating: null,
-          creator: "Charles",
-          participants: null,
-          date: "23 décembre 2018"
-        },
-        {
-          idActivity: 3,
-          idPlace: 3,
-          city: "Reims",
-          name: "World Championship LoL",
-          price: 20,
-          type: "soirée",
-          capacity: 10,
-          picture: "https://zupimages.net/up/18/48/luy5.jpg",
-          description:
-            "Pour ceux qui veulent voir les Worlds de LoL et boire un coup ensemble ! :)",
-          contact: "06.23.87.90.54",
-          rating: null,
-          creator: "Robin",
-          participants: null,
-          date: "14 décembre 2018"
-        }
-      ]
+      placeList: [],
+      activityList: [],
+      loading: true
     };
     this.changeViewToActivities = this.changeViewToActivities.bind(this);
     this.changeViewToPlaces = this.changeViewToPlaces.bind(this);
     this.toggle = this.toggle.bind(this);
   }
 
+  componentDidMount() {
+    this.callApiActivities();
+    this.callApiPlaces();
+    this.loading();
+  }
+
+  loading() {
+    setTimeout(() => {
+      this.setState({
+        loading: false
+      });
+    }, 3000);
+  }
+
+  callApiActivities() {
+    const activities = [];
+    axios
+      .get("http://localhost:3010/activities")
+      .then(response => response.data.map(index => activities.push(index)));
```

Comment:
what is the point of the activities array ? why don't you juste set the state when you receive the data ?